README.md

```diff
@@ -62,7 +62,29 @@ Check out [contribution guidelines](CONTRIBUTING.md) and other [awesome](https:/
 - [2007] [Innovation in Indie Games](https://www.youtube.com/watch?v=PR-ZtrOGHiY): Kyle Gabler, Jenova Chen, Jon Mak, Jon Blow (Panel)
 - [2000] [It's Great to be Back! Fast Code, Game Programming, and Other Thoughts from 20 (Minus Two) Years in the Trenches](http://www.gdcvault.com/play/1016641/): Michael Abrash (Rad Game Tools)
 
+### TED Talks
+
+[http://www.ted.com/topics/gaming](http://www.ted.com/topics/gaming)
+
+- [2015, TED]  [Physical therapy is boring — play a game instead](http://www.ted.com/talks/cosmin_mihaiu_physical_therapy_is_boring_play_a_game_instead): Cosmin Mihaiu (Indie) 
```

Comment:
This is nitpicky but could you make the talks title case (`Physical Therapy Is Boring — Play a Game Instead`)?